src/cmd/api/dto/json/jsonDTO.go

```diff
@@ -41,3 +41,19 @@ func VideoToStatusJson(video *models.Video) VideoStatus {
 
 	return videoStatus
 }
+
+// VideoInfo DTO
+
+type VideoInfo struct {
+	Title       string `json:"title" example:"amazingtitle"`
+	Upload_date string `json:"uploaddate" example:"January 2, 2006 15:04:05"`
```

Comment:
Are we sure about this format ? Why not use a more standard date format that can be understood by the frontend ?